func g() (int, error) { return 5, errorf("anError") }

x := g() // sloppy error gets transformed into a panic
